.github/workflows/ci.yml

```
@@ -34,61 +59,83 @@ jobs:
+            -   name: Checkout
+                uses: actions/checkout@v4
+                with:
+                    fetch-depth: 0
+
+            -   name: Download .NET coverage reports
+                uses: actions/download-artifact@v4
+                with:
+                    name: dotnet-coverage
+                    path: ./coverage/dotnet
+
+            -   name: Download Node.js coverage reports
```

YAMLlint (1.37.1)

[warning] too many spaces after hyphen

(hyphens)